cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java

```diff
@@ -95,6 +95,72 @@ public class ClouderaManagerSetupService implements ClusterSetupService {
 
     private static final String SDX_ENTERPRISE_DATALAKE_TEXT = "Enterprise SDX";
 
+    private static final String OPENSSL_INTERMEDIATE2018 =
```

Comment:
Can we move this constants to a separate constant class as it is almost 100 lines of Fedramp/Gov related configs?